### PR TITLE
Damage type standardization SKILL part 2/2

### PR DESCRIPTION
**Pull request by terrapin47**
Change damage types to Pierce/Slash/Blunt
Replacing instances of Hit, Strike, Impact, and Stab.

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -2412,7 +2412,7 @@ SKILL_20150317_002411	$Increases your movement speed by 10% per attribute level
 SKILL_20150317_002412	$Undistance: Physical Damage
 SKILL_20150317_002413	{memo X}Physical attack of allies within [Undistance] increases by 5% per attribute level.
 SKILL_20150317_002414	$Reconnaissance: Pierce Damage
-SKILL_20150317_002415	Monsters revealed with [Reconaissance] receive 10% more stab damage per level while the skill is enabled.
+SKILL_20150317_002415	Monsters revealed with [Reconaissance] receive 10% more pierce damage per level while the skill is enabled.
```
Comment (Sunaries) [inline SKILL.tsv]: pierce -> Pierce

File: SKILL.tsv [hunk 2]
```diff
@@ -4739,8 +4739,8 @@ SKILL_20150717_004738	{memo X}Max SP and Magic DEF increase based on number of [
 SKILL_20150717_004739	{memo X}Evasion increases based on number of [Leather] armor equipped and Attribute level. (at least 3 same equipment type).
 SKILL_20150717_004740	{memo X}Max HP and Physical DEF increase based on number of [Plate] armor equipped and Attribute level. (at least 3 same equipment type).
 SKILL_20150717_004741	Incurs additional 20 damage per Attribute level whenever attack from an enemy's back with dagger.
-SKILL_20150717_004742	$Increases the physical damage increase effect of [Gung Ho] by 1, while decreasing the decreased defense effect by 1 per attribute level.
-SKILL_20150717_004743	$Adds the AoE Attack Ratio of [Bash] by 1 per attribute level.
+SKILL_20150717_004742	$Increases the physical damage increase effect and the defense decrease effect of [Gung Ho] by 1 per attribute level
```
Comment (Sunaries) [inline SKILL.tsv]: add a period

File: SKILL.tsv [hunk 3]
```diff
@@ -4751,7 +4751,7 @@ SKILL_20150717_004750	$Adds the AoE Attack Ratio of [Wagon Wheel] by 1.
 SKILL_20150717_004751	$Deals additional damage equal to 50% physical attack to knock backed enemies with [Cartar Stroke].
 SKILL_20150717_004752	{memo X}The enemy that was attacked by [Skull Swing] will not be able to jump for 3 seconds.
 SKILL_20150717_004753	{memo X}The enemy that is attacked by [Pierce] will be in [Bleeding] for 5 seconds with 2% probability. Beeeding damage will be decided based on Physical ATK.
-SKILL_20150717_004754	$Decreases the Strike damage dealt on an enemy with [Synchro Thrusting] by 10%, while increasing the Pierce damage by 10% per attribute level.
+SKILL_20150717_004754	$Decreases the Blunt damage dealt on an enemy with [Synchro Thrusting] by 10%, while increasing the Pierce damage by 10% per attribute level.
```
Comment (Sunaries) [inline SKILL.tsv]: This is actually 'Hit' if we follow the guidelines of https://github.com/Treeofsavior/EnglishTranslation/issues/900

**Comment by terrapin47** [comment]
Added corrections based on feedback.

**Comment by Ragunaga** [comment]
I recommend changing blunt back to something similar to strike.

The other two types are Slash and Pierce, ways you can hit something. While blunt is more a description of the type of a weapon. You can still use a blunt weapon to slash and pierce.

Strike is more fitting and follows the theme of the other two.

**Comment by zhouyu47** [comment]
Does a "Strike weapon" make sense?  It doesn't work better both ways.  The word for blunt damage is bludgeon.

Strike doesn't immediately tell you what damage type it is.  Especially for someone playing a non-melee character.  For example, a wizard:
Enemies affected by [Lethargy] will receive 20% additional damage with X-type attacks.

If that X is Blunt, then it's clear that weapons like hammers, maces, and clubs will do more damage.  If that X is Strike, it's not clear what it's talking about.

**Pull request closed without merging**